app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
The `Elf` class should be abstract since it does not implement all abstract methods from the `Player` class.

Reply:
Why this not fixed ? Is Elf is not abstract class ? Since it has abstract method play_elf_song and passes test "test_main.py::test_classes_should_be_abstract[Elf] PASSED                [ 23%]"
![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/61278036/5846caf8-10b6-4d39-96c6-99e9ea7cdf56)

Reply:
Using provided definition from lesson - i can't get what isn't right!
![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/61278036/9a5bdb62-8899-497d-b368-15e670ffe868)